### PR TITLE
Safety improvements

### PR DESCRIPTION
**Pull request by kovmir**
Please accept my humble safety improvements to your `Makefile` and small polishes to `README.md`.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Set to true to enable experimental multithreading
-THREADS = false
+THREADS ?= false
```
Comment (sakhmatd) [inline makefile]: This should not be set if absent, this is really experimental and currently full of bugs.

**Review by sakhmatd — CHANGES_REQUESTED** [review]
Please address the changes in the review, good to commit except for one line.

**Review by sakhmatd — APPROVED** [review]

**Comment by sakhmatd** [comment]
Merged, thanks.

**Comment by kovmir** [comment]
Can you please make a new release with these changes? `2.0.1` supposedly.

**Comment by sakhmatd** [comment]
Done. There should also now be an automatic release triggered on every push to the master branch tagged "latest".

**Pull request merged**